Source/JavaScriptCore/heap/FullGCActivityCallback.cpp

```diff
@@ -41,7 +41,7 @@ void FullGCActivityCallback::doCollection(VM& vm)
     Heap& heap = vm.heap;
     m_didGCRecently = false;
 
-#if !PLATFORM(IOS_FAMILY) || PLATFORM(MACCATALYST)
+#if (!PLATFORM(IOS_FAMILY) && !PLATFORM(BROADCOM)) || PLATFORM(MACCATALYST)
```

Comment:
This should be a separate PR and we need to get feedback from other users. And if it is necessary, we need to think use WPE flag instead of a vendor.